pythonforandroid/bootstraps/common/build/templates/build.tmpl.gradle

```diff
@@ -13,68 +13,85 @@ allprojects {
     repositories {
```

Comment:
The changes here are much more minimal than I thought at first glance, due to the indentation changes.